docs/tutorial/avax-nft.md

```diff
@@ -0,0 +1,399 @@
+ChainID: 43112
+Symbol: AVAX
+Explorer: N/A
+
+> **Local testnet**: Listening port might not be 9650 depending on if you run an example program in `avalanchego` node or `avalanche-simulator` run script. The latter is recommended for quick start and will have a listening node on port 9650.
+
+Create a new Metamask account by importing this private key `0x56289e99c94b6912bfc12adc093c9b51124f0dc54ac7a766b2bc5ccf558d8027` on Metamask
+
+![adding new account on metamask](https://docs.avax.network/assets/images/Metamask-Import-Account-17b4d3c6e167ebf8709ace5bc30001f6.png)
+
+If all went well, you should have a funded Metamask AVAX wallet for building app locally:
+
+![Metamask with AVAX funded](https://i.imgur.com/fkLXV17.png)
+
+### Optional
+
+- Clone [Avalanche smart contract quickstart](https://github.com/ava-labs/avalanche-smart-contract-quickstart). Install deps with `yarn`,
+
+### Public API node
+
+Avalanche maintains a [public API gateway](https://docs.avax.network/build/tools/public-api), which you can use in quick development without having to run your own node.
+
+## Create a Node project
+
+With Node.js already installed, run `npm init` to create a new app project, then `cd` into your new directory and run the following:
+
+```shell
+npm install hardhat ethers @nomiclabs/hardhat-ethers --save-dev
+```
+Then, you can check if the installation was successful by typing `npx hardhat --version`. The Hard Hat CLI should print out the version number (yours may be different):
+
+```shell
+npx hardhat --version
+> 2.6.1
+```
+
+Now, with the local simulator nodes still running, run the following commands:
+
+```shell
+npx hardhat accounts --network local
+> 0x8db97C7cEcE249c2b98bDC0226Cc4C2A57BF52FC
+> 0x9632a79656af553F58738B0FB750320158495942
+> 0x55ee05dF718f1a5C1441e76190EB1a19eE2C9430
+> ...
+```
+
+Hopefully you should see a few addresses printed. Next, check the balances with:
+
+```shell
+npx hardhat balances --network local
+> 0x8db97C7cEcE249c2b98bDC0226Cc4C2A57BF52FC has balance 50000000000000000000000000
+> 0x9632a79656af553F58738B0FB750320158495942 has balance 0
+> 0x55ee05dF718f1a5C1441e76190EB1a19eE2C9430 has balance 0
+> ...
+```
+
+If you had correctly [created a keystore user and added test fund](#create-a-keystore-user-and-add-test-fund), you should see one of wealthy addresses shown.
+
+
+## Develop an NFT smart contract
+
+If you already have an existing smart contract, you may skip this section.
+
+We will create [ERC721](https://eips.ethereum.org/EIPS/eip-721) non-fungible tokens with their own attributes. To keep this simple, any account will be able to call a method `mintTo` to mint items.
+
+We will be using the standard [ERC721](https://docs.openzeppelin.com/contracts/4.x/erc721) smart contract from Open Zeppelin. Install it in your project with `npm install @openzeppelin/contract`.
+
+Create a directory named `/contracts` within the project root. Create a smart contract file named `Filet.sol` with the following code:
```

Comment:
Do developers have to create directories for `contracts`, `scripts` by themselves with hardhat? For truffle, we can run `truffle init` which will create the basic folder structure for a truffle smart contract project. Then developers can just create their files into the right folders.

Reply:
Actually hardhat gives several options when initializing the project, one being just creating a config file and another should be creating a project structure with `contracts` and `scripts` directories.